lib/api/LogManagerImpl.cpp

```diff
@@ -364,6 +364,8 @@ namespace MAT_NS_BEGIN
 
     void LogManagerImpl::FlushAndTeardown()
     {
+        PauseActivity();
```

Comment:
1. If we are already calling PauseActivity and WaitPause in FlushAndTeardown, do we need to expose wrapper class methods?
2. Since flushAndTeardown is being called in other scenarios except for app termination, is it safe to have PauseActivity and WaitPause here?
cc: @aamittal1989 @karthika-msft

Reply:
I can see `PauseActivity` being useful independently as well. For example, you can pause the logmanager if you know the app is going to suspend and try to resume later. You can also pause for, say, when the device is roaming or in low power mode and resume when things are normal.